lblod/app-subsidiepunt
# Feature/e inclusion

## ID
DGS-371

## Description
This PR updates config and adds missing rdf:types so that e-inclusion verantwoording step data gets produced for subsidiedatabank

## Type of change


## How to setup
- drc up


## How to test
- access frontend
- create new e-inclusion step and fill in everything
- either check publication graph to see if all data gets produced or setup subsidiedatabank and check if it consumed all data and has it available in the frontend
- to check frontend subsidiedatabank, checkout following PR: https://github.com/lblod/frontend-subsidiedatabank/pull/32
